src/data/finset/powerset.lean

```diff
@@ -222,17 +222,28 @@ begin
     simp }
 end
 
-lemma powerset_card_bUnion [decidable_eq (finset α)] (s : finset α) :
-  finset.powerset s = (range (s.card + 1)).bUnion (λ i, powerset_len i s) :=
+lemma pairwise_disjoint_powerset_len (s : finset α) :
+  _root_.pairwise (λ i j, disjoint (s.powerset_len i) (s.powerset_len j)) :=
+λ i j hij, finset.disjoint_left.mpr $ λ x hi hj, hij $
+  (mem_powerset_len.mp hi).2.symm.trans (mem_powerset_len.mp hj).2
```

Comment:
I am rather sure that this lemma already exists, although I cannot find it on the fly.

Maybe it's because I have it on a branch 🤔

Reply:
It exists for multisets: https://leanprover-community.github.io/mathlib_docs/data/multiset/powerset.html#multiset.disjoint_powerset_len

Should I drop the `pairwise` statement to match that?

Reply:
I prefer your spelling, in fact. I think you can change the `multiset` one in this PR, because the resulting lemma is defeq so not much should break.

Reply:
Done